portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +24,24 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
+  
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    /*response.setContentType("text/html;");
+    response.getWriter().println("<h1>Hello Oyin!</h1>");*/
+
+    ArrayList<String> comments = new ArrayList<String>();
+    comments.add("How do I sign up to go next year?");
+    comments.add("Is Grace Hopper a real person?");
+    comments.add("Wow! I bet you got a lot of swag!lol ");
+   
+    response.setContentType("application/json;");
+    response.getWriter().println(convertToJsonUsingGson(comments));
+
+  }
+  private String convertToJsonUsingGson(ArrayList comments){
+    Gson gson = new Gson();
+    String json = gson.toJson(comments);
+    return json;
```

Comment:
Nit: the variable `json` is not necessary.
```suggestion
    return gson.toJson(comments);
```